redis/quickstarts/nextjsapprouter.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "App Router"
+---
+
+### Project Setup
+
+Let's create a new Next.js application with App Router and install `@upstash/redis` package.
+
+```shell
+npx create-next-app@latest
+cd my-app
+npm install @upstash/redis
+```
+
+### Database Setup
+
+Create a Redis database using [Upstash Console](https://console.upstash.com) or [Upstash CLI](https://github.com/upstash/cli) and copy the `UPSTASH_REDIS_REST_URL` and `UPSTASH_REDIS_REST_TOKEN` into your `.env` file.
+
+```shell .env
+UPSTASH_REDIS_REST_URL=<YOUR_URL>
+UPSTASH_REDIS_REST_TOKEN=<YOUR_TOKEN>
+```
+
+### Home Page Setup
+
+Update `/app/page.tsx`:
+```tsx /app/page.tsx
+import { Redis } from "@upstash/redis";
```

Comment:
Don't we have to specify its "use server" here?

Reply:
I think Next.js uses Server Components by default in App Router so this should work fine. If you think there is a need for or a benefit in using Server Actions I can change.

Reply:
Nah. I'm just making sure.